app/code/Magento/WishlistGraphQl/Model/Resolver/Wishlist/AddToCart.php

```diff
@@ -0,0 +1,231 @@
+        AddProductsToCartService $addProductsToCart,
+        CartItemsRequestBuilder $cartItemsRequestBuilder,
+        Uid $uidEncoder
+    ) {
+        $this->wishlistResource = $wishlistResource;
+        $this->wishlistFactory = $wishlistFactory;
+        $this->wishlistConfig = $wishlistConfig;
+        $this->addProductsToWishlist = $addProductsToWishlist;
+        $this->wishlistDataMapper = $wishlistDataMapper;
+        $this->quantityProcessor = $quantityProcessor;
+        $this->createEmptyCartForCustomer = $createEmptyCartForCustomer;
+        $this->addProductsToCartService = $addProductsToCart;
+        $this->cartItemsRequestBuilder = $cartItemsRequestBuilder;
+        $this->uidEncoder = $uidEncoder;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function resolve(
+        Field $field,
+        $context,
+        ResolveInfo $info,
+        array $value = null,
+        array $args = null
+    ) {
+        if (!$this->wishlistConfig->isEnabled()) {
+            throw new GraphQlInputException(__('The wishlist configuration is currently disabled.'));
+        }
+
+        $customerId = $context->getUserId();
+
+        /* Guest checking */
+        if (null === $customerId || 0 === $customerId) {
+            throw new GraphQlAuthorizationException(__('The current user cannot perform operations on wishlist'));
+        }
+
+        if (empty($args['wishlistUid'])) {
+            throw new GraphQlInputException(__('"wishlistUid" value should be specified'));
+        }
+        $wishlistId = (int) $this->uidEncoder->decode($args['wishlistUid']);
+        $wishlist = $this->getWishlist($wishlistId, $customerId);
+
+        if (null === $wishlist->getId() || $customerId !== (int) $wishlist->getCustomerId()) {
+            throw new GraphQlInputException(__('The wishlist was not found.'));
+        }
+
+        $itemIds = [];
+        if (isset($args['wishlistItemUids'])) {
+            $itemIds = array_map(
+                function ($id) {
+                    return $this->uidEncoder->decode($id);
+                },
+                $args['wishlistItemUids']
+            );
+        }
+
+        $collection = $this->getWishlistItems($wishlist, $itemIds);
+
+        $maskedCartId = $this->createEmptyCartForCustomer->execute($customerId);
+
+        $cartItems = [];
+        foreach ($collection as $item) {
+            $disableAddToCart = $item->getProduct()->getDisableAddToCart();
+            $item->getProduct()->setDisableAddToCart($disableAddToCart);
+            $cartItemsData = $this->cartItemsRequestBuilder->build($item);
+            foreach ($cartItemsData as $cartItemData) {
+                $cartItems[] = (new CartItemFactory())->create($cartItemData);
+            }
+        }
+
+        /** @var AddProductsToCartOutput $addProductsToCartOutput */
+        $addProductsToCartOutput = $this->addProductsToCartService->execute($maskedCartId, $cartItems);
+
```

Comment:
How are we removing the items from the wishlist after it is added to the cart?

Reply:
Actually not removing wishlist items now. I missed that part. I will push changes for this today